docs/UserGuide.md

```diff
@@ -127,6 +127,20 @@ Examples:
 * `delstaff pn/2103 sn/John` Delete staff member named `John` from project `2103`.
 * `delstaff pn/DUKE sn/Betty` Delete staff member named `Betty` from project `DUKE`.
 
```

Comment:
Might need to reconsider the way we use our delete staff command. Would delstaff<index> be better? But means that we can only call delstaff after a view command.